jobs/pacman-cloud-discovery/src/main/java/com/tmobile/cso/pacman/inventory/file/AssetFileGenerator.java

```diff
@@ -1233,7 +1234,7 @@ public void generateFiles(List<Map<String,String>> accounts,String skipRegions,S
 		ErrorManageUtil.omitOpsAlert();
 		if(!ErrorManageUtil.getErrorMap().isEmpty()){
 			//Below logger message is used by datadog to create notification in slack
-			log.error("Error occurred in atleast one collector for jobId : AWS-Data-Collector-Job");
+			log.error("Error occurred in job aws-data-collector-job in at least one collector.");
```

Comment:
same as two other logs

Reply:
This is needed. Hence the change. Comment in line 1236 mentions the actual purpose of the logger statement.

Reply:
please use the new log format from above comments